src/EnumExamples.cs

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2019 Henry Gabryjelski
```

Comment:
Do we have to enforce a copyright?

Reply:
Copyrights exist even if not marked.  License still applies.